.credo.exs

```diff
@@ -87,7 +87,7 @@
         {Credo.Check.Readability.MaxLineLength, priority: :low, max_length: 120},
         {Credo.Check.Readability.ModuleAttributeNames},
         {Credo.Check.Readability.ModuleDoc, false},
-        {Credo.Check.Readability.AliasOrder},
+        {Credo.Check.Readability.AliasOrder, false},
```

Comment:
Is there any reason to disable this check?

Reply:
It didn’t seem to add much value to me. Happy to put it back though!

Reply:
Might be worth leaving it as is for now and changing in a separate PR in case anyone has any strong feelings.